### PR TITLE
LazyBin fix, pull binaries only when they are ran.

### PR DESCRIPTION
**Pull request by sugarraysam**
# py-mtcli

## Description

Delay pulling sretoolbox binaries to the moment when they are needed.

### CODE REVIEW

File: managedtenants/bundles/binary_deps.py [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+    def run(self, cmd):
+        if self.instance is None:
+            self.instance = self.bin_class(self.version, self.download_path)
+
+        self.instance.run(*cmd)
+
+
+OPM = LazyBin(Opm, version="1.19.5", download_path="/tmp")
+MTCLI = LazyBin(Mtcli, version="0.10.0", download_path="/tmp")
+OPERATOR_SDK = LazyBin(OperatorSDK, version="1.4.2", download_path="/tmp")
```
Comment (sandhyadalavi) [inline managedtenants/bundles/binary_deps.py:25]: can we extract out these versions from here?

Comment (sandhyadalavi) [reply managedtenants/bundles/binary_deps.py:25]: other than that looks good

Comment (sugarraysam) [reply managedtenants/bundles/binary_deps.py:25]: I am sorry but I am not sure I understand. What do you mean?

Comment (sandhyadalavi) [reply managedtenants/bundles/binary_deps.py:25]: can we not hard-code these version here? extract out in a config file?

Comment (sugarraysam) [reply managedtenants/bundles/binary_deps.py:25]: I think extracting those versions to a config file, and giving a user the ability to change them at runtime is dangerous. The `managedtenants/bundles` package depends on these versions, and is only tested for these versions. I think we need to keep them hardcoded, and test any change with the whole PR->review->release process.

**Pull request merged**